File: projects/project10/Java/Ostrich.java
/* Ostrich.java models a Ostrich
 *
 * Created by: Ethan Clark Project10
 * Date: April 25, 2016
 */

public class Ostrich extends WalkingBird
{

	/**********************************************
	* Constructor for the child Ostrich class     *
	* Sets name to be myName in the parent class  *
	**********************************************/
	public Ostrich(String name)
	{
		super(name);
	}

	/*************************************************
	* call() returns the sound for the Ostrich class *
	* Receive: NONE                                  *
	* Return: "Snork!"                               *
	*************************************************/
	public String call()
	{
		return "Snork!";
	}
}

File: projects/project10/Ruby/Ostrich.rb
# Ostrich.rb | Defines the Ostrich class which inherits attributes and methods
#   from the WalkingBird superclass.
#
# Created by: Ethan Clark Project10
# Date: April 26, 2016
####################################################

require './WalkingBird.rb'

class Ostrich < WalkingBird

	######################################################
	# call returns the sound that a Ostrich type makes   #
	# Recieve: NONE                                      #
	# Return: Snork!, a String                           #
	######################################################
	def call
		"Snork!"
	end

end
